cmd/schema-gen/main.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2019
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package main
+
+// NOT part of the KUDO distribution
```

Comment:
should we put this into hack as well?I think it's less confusing that have it in cmd when it's not really relevant for the project that much, it's rather some tooling...

Reply:
I'd lean against putting more Go into `hack`, I'd rather have maybe a `kudo-gen` repo with this and any other generative tools and depend on it rather than moving it into hack. Generally ok with moving it out of this repo's `cmd/` though. ;)

Reply:
slept on this, I'd be ok with `hack/cmd`